python/mspasspy/db/database.py

```diff
@@ -12,31 +11,55 @@
 
 import bson.errors
 import bson.objectid
+import dask.bag as daskbag
+from pymongo import MongoClient
```

Comment:
Actually, we should just use our own `Client` instead of the `MongoClient` in this module.

Reply:
I just tried to use our Client but it will cause the "Circular Import Error" since our client file needs to use the Database. I think we can keep using the pymongo client here since it doesn't affect the behavior.

Reply:
Good catch... such circular dependency usually means that we didn't arrange the modules correctly. Because this `MongoClient` is introduced by `read_distributed_data` maybe we can consider relocate this function. Also, a less elegant way around it is to import `Client` within the `_read_distributed_data` function.

Reply:
done